Cannot pack charm

**Describe the bug**
command `make charm` and `charmcraft pack` both fail with no useful traceback
**To Reproduce**
Steps to reproduce the behavior:
1. Run either command in a terminal
2. See 
    ```Failed to build charm for bases index '0'.                                                                              
    Full execution log: '/home/pietro/snap/charmcraft/common/cache/charmcraft/log/charmcraft-20220214-123508.544422.log'    
    make: *** [Makefile:19: charm] Error 1
    ```
<details><summary>charmcraft-20220214-123508.544422.log</summary>
<p>

    2022-02-14 12:35:08.547 Starting charmcraft version 1.4.0
    2022-02-14 12:35:08.548 Raw pre-parsed sysargs: args={'help': False, 'verbose': False, 'quiet': False, 'trace': False, 'project_dir': None} filtered=['pack']
    2022-02-14 12:35:08.548 General parsed sysargs: command='pack' args=[]
    2022-02-14 12:35:08.549 Command parsed sysargs: Namespace(bases_index=None, debug=False, destructive_mode=False, entrypoint=None, force=False, requirement=None, shell=False, shell_after=False)
    2022-02-14 12:35:08.549 Packing the charm.
    2022-02-14 12:35:08.549 Working arguments: {'from': PosixPath('/home/pietro/canonical/charm-prometheus-node-exporter'), 'destructive_mode': False, 'entrypoint': None, 'requirement': [], 'bases_indices': None, 'force': False, 'debug': False, 'shell': False, 'shell_after': False}
    2022-02-14 12:35:08.549 Parsing '/home/pietro/canonical/charm-prometheus-node-exporter/metadata.yaml'
    2022-02-14 12:35:08.641 Executing on host: lxc --project default profile show local:default
    2022-02-14 12:35:08.704 Building for 'bases[0]' as host matches 'build-on[0]'.
    2022-02-14 12:35:08.705 Building for 'bases[0][0]'.
    2022-02-14 12:35:08.705 Launching environment to pack for base name='ubuntu' channel='20.04' architectures=['amd64']
    2022-02-14 12:35:08.705 Executing on host: lxc remote list --format=yaml
    2022-02-14 12:35:08.769 Remote 'craft-com.ubuntu.cloud-buildd' already exists.
    2022-02-14 12:35:08.769 Executing on host: lxc project list local: --format=yaml
    2022-02-14 12:35:08.835 Executing on host: lxc --project charmcraft list local: --format=yaml
    2022-02-14 12:35:08.968 Executing on host: lxc --project charmcraft list local: --format=yaml
    2022-02-14 12:35:09.103 Executing on host: lxc --project charmcraft start local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64
    2022-02-14 12:35:09.442 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 cat /etc/os-release
    2022-02-14 12:35:09.523 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 test -f /etc/craft-instance.conf
    2022-02-14 12:35:09.644 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 cat /etc/craft-instance.conf
    2022-02-14 12:35:09.764 Executing on host: lxc --project charmcraft file push /home/pietro/snap/charmcraft/common/tmpr3uxgskm local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/etc/environment --mode=0644
    2022-02-14 12:35:09.833 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 chown root:root /etc/environment
    2022-02-14 12:35:09.947 Waiting for environment to be ready...
    2022-02-14 12:35:09.947 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl is-system-running
    2022-02-14 12:35:10.083 Executing on host: lxc --project charmcraft file push /home/pietro/snap/charmcraft/common/tmpbbvogvy1 local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/etc/craft-instance.conf --mode=0644
    2022-02-14 12:35:10.152 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 chown root:root /etc/craft-instance.conf
    2022-02-14 12:35:10.264 Executing on host: lxc --project charmcraft file push /home/pietro/snap/charmcraft/common/tmp8_9_nkuv local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/etc/hostname --mode=0644
    2022-02-14 12:35:10.334 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 chown root:root /etc/hostname
    2022-02-14 12:35:10.452 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 hostname -F /etc/hostname
    2022-02-14 12:35:10.594 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 ln -sf /run/systemd/resolve/resolv.conf /etc/resolv.conf
    2022-02-14 12:35:10.718 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl enable systemd-resolved
    2022-02-14 12:35:10.948 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl restart systemd-resolved
    2022-02-14 12:35:11.275 Executing on host: lxc --project charmcraft file push /home/pietro/snap/charmcraft/common/tmpkgkwxem0 local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/etc/systemd/network/10-eth0.network --mode=0644
    2022-02-14 12:35:11.351 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 chown root:root /etc/systemd/network/10-eth0.network
    2022-02-14 12:35:11.472 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl enable systemd-networkd
    2022-02-14 12:35:11.676 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl restart systemd-networkd
    2022-02-14 12:35:12.095 Waiting for networking to be ready...
    2022-02-14 12:35:12.095 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 getent hosts snapcraft.io
    2022-02-14 12:35:12.252 Executing on host: lxc --project charmcraft file push /home/pietro/snap/charmcraft/common/tmpcoxqnguv local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/etc/apt/apt.conf.d/00no-recommends --mode=0644
    2022-02-14 12:35:12.323 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 chown root:root /etc/apt/apt.conf.d/00no-recommends
    2022-02-14 12:35:12.446 Executing on host: lxc --project charmcraft file push /home/pietro/snap/charmcraft/common/tmp3jwh_6vu local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/etc/apt/apt.conf.d/00update-errors --mode=0644
    2022-02-14 12:35:12.515 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 chown root:root /etc/apt/apt.conf.d/00update-errors
    2022-02-14 12:35:12.625 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 apt-get update
    2022-02-14 12:35:13.725 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 apt-get install -y apt-utils
    2022-02-14 12:35:14.236 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 apt-get install -y fuse udev
    2022-02-14 12:35:14.779 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl enable systemd-udevd
    2022-02-14 12:35:14.992 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl start systemd-udevd
    2022-02-14 12:35:15.131 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 apt-get install -y snapd
    2022-02-14 12:35:15.569 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl start snapd.socket
    2022-02-14 12:35:15.691 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 systemctl restart snapd.service
    2022-02-14 12:35:15.857 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 snap wait system seed.loaded
    2022-02-14 12:35:15.980 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 rm -f /tmp/charmcraft.snap
    2022-02-14 12:35:16.207 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 test -d /tmp
    2022-02-14 12:35:16.325 Executing on host: lxc --project charmcraft file push /home/pietro/tmpyuzrflks.tmp-craft/charmcraft.snap local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/tmp/charmcraft.snap --gid=0 --uid=0
    2022-02-14 12:35:16.531 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 snap install /tmp/charmcraft.snap --classic --dangerous
    2022-02-14 12:35:18.973 Executing on host: lxc --project charmcraft config device show local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64
    2022-02-14 12:35:19.049 Executing on host: lxc --project charmcraft config device add local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 disk-/root/project disk source=/home/pietro/canonical/charm-prometheus-node-exporter path=/root/project
    2022-02-14 12:35:19.142 Packing the charm
    2022-02-14 12:35:19.142 Running ['charmcraft', 'pack', '--bases-index', '0']
    2022-02-14 12:35:19.142 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 --cwd /root/project -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 charmcraft pack --bases-index 0
    2022-02-14 12:35:31.968 :: Packing the charm.                                                             
    Building charm in '/root'                                                      
    Building charm in '/root' - (2.0s)                                             
    Building charm in '/root' \ (2.1s)                                             
    Building charm in '/root' | (2.2s)                                             
    Building charm in '/root' / (2.3s)                                             
    Building charm in '/root' - (2.4s)                                             
    Building charm in '/root' \ (2.5s)                                             
    Building charm in '/root' | (2.6s)                                             
    Building charm in '/root' / (2.7s)                                             
    Building charm in '/root' - (2.8s)                                             
    Building charm in '/root' \ (2.9s)                                             
    Building charm in '/root' | (3.0s)                                             
    Building charm in '/root' / (3.1s)                                             
    Building charm in '/root' - (3.2s)                                             
    Building charm in '/root' \ (3.3s)                                             
    Building charm in '/root' | (3.4s)                                             
    Building charm in '/root' / (3.5s)                                             
    Building charm in '/root' - (3.6s)                                             
    Building charm in '/root' \ (3.7s)                                             
    Building charm in '/root' | (3.8s)                                             
    Building charm in '/root' / (3.9s)                                             
    Building charm in '/root' - (4.0s)                                             
    Building charm in '/root' \ (4.1s)                                             
    Building charm in '/root' | (4.2s)                                             
    Building charm in '/root' / (4.3s)                                             
    Building charm in '/root' - (4.4s)                                             
    Building charm in '/root' \ (4.5s)                                             
    Building charm in '/root' | (4.6s)                                             
    Building charm in '/root' / (4.7s)                                             
    Building charm in '/root' - (4.8s)                                             
    Building charm in '/root' \ (4.9s)                                             
    Building charm in '/root' | (5.0s)                                             
    Building charm in '/root' / (5.1s)                                             
    Building charm in '/root' - (5.2s)                                             
    Building charm in '/root' \ (5.3s)                                             
    Building charm in '/root' | (5.4s)                                             
    Building charm in '/root' / (5.5s)                                             
    Building charm in '/root' - (5.6s)                                             
    Building charm in '/root' \ (5.7s)                                             
    Building charm in '/root' | (5.8s)                                             
    Building charm in '/root' / (5.9s)                                             
    Building charm in '/root' - (6.0s)                                             
    Building charm in '/root' \ (6.1s)                                             
    Building charm in '/root' | (6.2s)                                             
    Building charm in '/root' / (6.3s)                                             
    Building charm in '/root' - (6.4s)                                             
    Building charm in '/root' \ (6.5s)                                             
    Building charm in '/root' | (6.6s)                                             
    Building charm in '/root' / (6.7s)                                             
    Building charm in '/root' - (6.8s)                                             
    Building charm in '/root' \ (6.9s)                                             
    Building charm in '/root' | (7.0s)                                             
    Building charm in '/root' / (7.1s)                                             
    Building charm in '/root' - (7.2s)                                             
    Building charm in '/root' \ (7.3s)                                             
    Building charm in '/root' | (7.4s)                                             
    Building charm in '/root' / (7.5s)                                             
    Building charm in '/root' - (7.6s)                                             
    Building charm in '/root' \ (7.7s)                                             
    Building charm in '/root' | (7.8s)                                             
    Building charm in '/root' / (7.9s)                                             
    Building charm in '/root' - (8.0s)                                             
    Building charm in '/root' \ (8.1s)                                             
    Building charm in '/root' | (8.2s)                                             
    Building charm in '/root' / (8.3s)                                             
    Building charm in '/root' - (8.4s)                                             
    Building charm in '/root' \ (8.5s)                                             
    Building charm in '/root' | (8.6s)                                             
    Building charm in '/root' / (8.7s)                                             
    Building charm in '/root' - (8.8s)                                             
    Building charm in '/root' \ (8.9s)                                             
    Building charm in '/root' | (9.0s)                                             
    Building charm in '/root' / (9.1s)                                             
    Building charm in '/root' - (9.2s)                                             
    Building charm in '/root' \ (9.3s)                                             
    Building charm in '/root' | (9.4s)                                             
    Building charm in '/root' / (9.5s)                                             
    Building charm in '/root' - (9.6s)                                             
    Building charm in '/root' \ (9.7s)                                             
    Building charm in '/root' | (9.8s)                                             
    Building charm in '/root' / (9.9s)                                             
    Building charm in '/root' - (10.0s)                                            
    Building charm in '/root' \ (10.1s)                                            
    Building charm in '/root' | (10.2s)                                            
    Building charm in '/root' / (10.3s)                                            
    Building charm in '/root' - (10.4s)                                            
    Building charm in '/root' \ (10.5s)                                            
    Building charm in '/root' | (10.6s)                                            
    Building charm in '/root' / (10.7s)                                            
    Building charm in '/root' - (10.8s)                                            
    Building charm in '/root' \ (10.9s)                                            
    Building charm in '/root' | (11.0s)                                            
    Building charm in '/root'                                                      
    Parts processing error: Failed to run the build script for part 'charm'.
    2022-02-14 12:35:31.971 :: Full execution log: '/root/snap/charmcraft/common/cache/charmcraft/log/charmcraft-20220214-113520.844195.log'
    2022-02-14 12:35:32.071 Executing in container: lxc --project charmcraft exec local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 -- env PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin CHARMCRAFT_MANAGED_MODE=1 test -f /tmp/charmcraft.log
    2022-02-14 12:35:32.201 Executing on host: lxc --project charmcraft file pull local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64/tmp/charmcraft.log /home/pietro/snap/charmcraft/common/charmcraft-mps9dfgb
    2022-02-14 12:35:32.283 Logs captured from managed instance:
    2022-02-14 12:35:32.283 Executing on host: lxc --project charmcraft config device show local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64
    2022-02-14 12:35:32.359 Executing on host: lxc --project charmcraft config device remove local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64 disk-/root/project
    2022-02-14 12:35:32.460 Executing on host: lxc --project charmcraft stop local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64
    2022-02-14 12:35:33.781 Failed to build charm for bases index '0'.
    2022-02-14 12:35:33.783 Traceback (most recent call last):
    2022-02-14 12:35:33.783   File "/snap/charmcraft/761/lib/charmcraft/commands/build.py", line 467, in pack_charm_in_instance
    2022-02-14 12:35:33.783     instance.execute_run(
    2022-02-14 12:35:33.783   File "/snap/charmcraft/761/lib/craft_providers/lxd/lxd_instance.py", line 221, in execute_run
    2022-02-14 12:35:33.783     return self.lxc.exec(
    2022-02-14 12:35:33.783   File "/snap/charmcraft/761/lib/craft_providers/lxd/lxc.py", line 307, in exec
    2022-02-14 12:35:33.783     return runner(final_cmd, **kwargs)  # pylint: disable=subprocess-run-check
    2022-02-14 12:35:33.783   File "/snap/charmcraft/761/usr/lib/python3.8/subprocess.py", line 516, in run
    2022-02-14 12:35:33.783     raise CalledProcessError(retcode, process.args,
    2022-02-14 12:35:33.783 subprocess.CalledProcessError: Command '['lxc', '--project', 'charmcraft', 'exec', 'local:charmcraft-prometheus-node-exporter-20977039-0-0-amd64', '--cwd', '/root/project', '--', 'env', 'PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/snap/bin', 'CHARMCRAFT_MANAGED_MODE=1', 'charmcraft', 'pack', '--bases-index', '0']' returned non-zero exit status 1.
    2022-02-14 12:35:33.783 Full execution log: '/home/pietro/snap/charmcraft/common/cache/charmcraft/log/charmcraft-20220214-123508.544422.log'
</p>
</details>

**Desktop:**
 - OS and Version: Ubuntu 20.04
 - charmcraft version: 1.4.0    
 - Charm version: main

--- a/charmcraft.yaml
+++ b/charmcraft.yaml
@@ -10,3 +10,7 @@ bases:
       - name: centos
         channel: "7"
         architectures: [amd64]
+parts:
+  charm:
+    build-packages: [git]
+    charm-python-packages: [setuptools]

